### DISCUSSION TITLE
Spent two minutes in gimp and made this. Whadd'yall think?

### DISCUSSION THREAD

**Discussion by 2DSBrewer** [kth5/archpower#84] [General]
![arch-power-linux](https://github.com/kth5/archpower/assets/86933537/fa7ea8b4-ca42-4e8d-ba45-4d698e190afa)

**Comment by kth5** [comment]
Cool beans!

Not sure this will infringe on any trademarks, it looks very much like the original PowerPC logo. Perhaps we can at least not colour it red?

Original Logo on the Wikipedia page for PowerPC:
https://en.wikipedia.org/wiki/PowerPC

Same goes for the Archlinux logo, without their permission it may not be OK to use it verbatim.

**Reply by 2DSBrewer** [reply]
Yeah, you are probably right, I will try to make another one that's less likely to make IBM and Arch annoyed.
